app/controllers/portal_controller.rb

```diff
@@ -97,6 +97,15 @@ def galleries
     end
   end
 
+  def parent
+    # Search the API for the record with dcterms:hasPart data.europeana.eu/item/RECORD_ID
+    @parent = search_results_for_dcterms_has_part(doc_id)[:items]&.first
+
+    respond_to do |format|
+      format.json { render :parent, layout: false }
```

Comment:
should we be concerned with a case where the parent couldn't be found? For example for a record that says it's part _of some record that doesn't contain that record as a hasPart.